src/components/Button/Button.tsx

```diff
@@ -39,13 +40,19 @@ const Button = React.forwardRef<HTMLButtonElement, ButtonProps>((props, ref) =>
   return (
     <ButtonBase {...props} ref={ref} isLoading={isLoading} onClick={onClick}>
       <span css={buttonSpanStyle()}>
-        {hasAvatar && <Avatar src={avatar?.src}>{avatar?.label}</Avatar>}
-        {iconLeftName && !hasAvatar && (
-          <Icon name={iconLeftName} color={tokens(`${type}.textColor` as ButtonTokens)} />
-        )}
-        <span>{children}</span>
-        {iconRightName && (
-          <Icon name={iconRightName} color={tokens(`${type}.textColor` as ButtonTokens)} />
+        {size === 'compact' ? (
```

Comment:
compact button doesnt have icon or avatar? this is really different.

I would expect documentation to reflect this as well as types

Reply:
according to figma docs `‘compact’ variant only supports simple text button (no avatars, icons), icon buttons, and dropdown buttons`.

I will add that to storybook also